tests/functional/spec/placeholder_answers_on_the_path.spec.js

```diff
@@ -0,0 +1,111 @@
+import DateEntryBlockPage from "../generated_pages/placeholder_first_non_empty_item/date-entry-block.page";
+import DateQuestionBlockPage from "../generated_pages/placeholder_first_non_empty_item/date-question-block.page";
+import TotalTurnoverBlockPage from "../generated_pages/placeholder_first_non_empty_item/total-turnover-block.page";
+import FoodQuestionBlockPage from "../generated_pages/placeholder_first_non_empty_item_cross_section_dependencies/food-question-block.page";
+
+import AddPersonPage from "../generated_pages/placeholder_first_non_empty_item_repeating_sections/list-collector-add.page";
+import ListCollectorPage from "../generated_pages/placeholder_first_non_empty_item_repeating_sections/list-collector.page";
+import RepeatingSectionPage from "../generated_pages/placeholder_first_non_empty_item_repeating_sections/repeating-section-summary.page";
+import HubPage from "../base_pages/hub.page.js";
+
+describe("First Non Empty Item Transform", () => {
+  before("Launch survey", async () => {
+    await browser.openQuestionnaire("test_placeholder_first_non_empty_item.json");
+  });
+
+  it("When the custom date range is entered and the answer is changed back to metadata date range, Then metadata date should be displayed", async () => {
+    // Set the date
+    await $(DateQuestionBlockPage.noINeedToReportForADifferentPeriod()).click();
+    await $(DateQuestionBlockPage.submit()).click();
+    await $(DateEntryBlockPage.dateEntryFromday()).setValue("5");
+    await $(DateEntryBlockPage.dateEntryFrommonth()).setValue("01");
+    await $(DateEntryBlockPage.dateEntryFromyear()).setValue("2017");
+    await $(DateEntryBlockPage.dateEntryToday()).setValue("25");
+    await $(DateEntryBlockPage.dateEntryTomonth()).setValue("01");
+    await $(DateEntryBlockPage.dateEntryToyear()).setValue("2017");
+    await $(DateEntryBlockPage.submit()).click();
+    // Change to original dates
+    await $(TotalTurnoverBlockPage.previous()).click();
+    await $(DateEntryBlockPage.previous()).click();
+    await $(DateQuestionBlockPage.yesICanReportForThisPeriod()).click();
+    await $(DateQuestionBlockPage.submit()).click();
+    await expect(await browser.getUrl()).to.contain(TotalTurnoverBlockPage.pageName);
+    await expect(await $(TotalTurnoverBlockPage.questionTitle()).getText()).to.contain("1 January 2017 to 1 February 2017");
+  });
+});
+
+describe("First Non Empty Item Transform Cross Section", () => {
+  before("Launch survey", async () => {
+    await browser.openQuestionnaire("test_placeholder_first_non_empty_item_cross_section_dependencies.json");
+    await $(HubPage.submit()).click();
+  });
+
+  it("When the custom date range is entered and the answer is changed back to metadata range, then metadata should be displayed for both sections", async () => {
```

Comment:
Should be Given When Then
```suggestion
  it("Given a custom date range is entered, When the answer is changed back to metadata range, Then the metadata date should be displayed for both sections", async () => {
```